brut.apktool/apktool-lib/src/main/java/brut/androlib/BackgroundWorker.java

```diff
@@ -0,0 +1,65 @@
+package brut.androlib;
+
+import java.util.ArrayList;
+import java.util.concurrent.*;
+
+public class BackgroundWorker {
+
+    private static final int THREADS_COUNT = Runtime.getRuntime().availableProcessors();
```

Comment:
Should this be allowed to be changed via like `--jobs`? It seems common in multi-threaded tools to control the amount of threads.

Reply:
No problem. Should we then delete `BackgroundWorker(int threads)` method?

Reply:
> Should this be allowed to be changed via like `--jobs`? It seems common in multi-threaded tools to control the amount of threads.

No problem. Should we then delete `BackgroundWorker(int threads)` method?